recovery.md

```diff
@@ -0,0 +1,214 @@
+# Building Poplar Debian System Recovery Media From Source
+
+[![Creative Commons Licence](https://licensebuttons.net/l/by-sa/4.0/88x31.png)](http://creativecommons.org/licenses/by-sa/4.0/)
+
+The instructions that follow describe the process for creating a USB
+flash drive suitable for use in recovering a Poplar system from a
+"bricked" state.  The USB memory stick must be at least 2 GB.
+
+**NOTE:** To create recovery media from pre-built images rather than
```

Comment:
Are we suppose to use this file for both android and debian recovery?

Reply:
Yes, it can be used for both Android and Debian.

Reply:
Android and Debian are using different partition table though, and current debian is using tftp but android use usb.
There are still some work here..

Reply:
Hrm.. ok. Sorry I didn't know that. Let me check and post and update.

Reply:
So, the prebuilt files in `https://github.com/96boards-poplar/l-loader/tree/master/installer/recovery_files` are Android-specific only? In that case, I'll remove this paragraph until we have pre-built recovery files for Debian in `l-loader`. Agree?

Reply:
That's for Android only.

Reply:
@shawnguo2 Is it possible for you to push Debian prebuilt files to `l-loader` as well? I can try to do it but not sure if there are any special requirements. Or you can let me know the requirements. Then Debian users can have the option to use prebuilt files for recovery.